packages/server/test/support/fixtures/projects/e2e/cypress/integration/various_failures_spec.js

```diff
@@ -0,0 +1,409 @@
+/**
```

Comment:
TODO: Add failures with `cy.each` and other commands that takes a callback function.